src/v2/view-builder/utils/FactorUtil.js

```diff
@@ -45,14 +41,13 @@ const factorData = {
   },
```

Comment:
@magizh-okta line #38 - I have not removed it like other authenticators, so the label is hard-coded to come from the props file. Want me to change?

Reply:
we want to override it can you please put back the label here.

Reply:
I guess Haisheng meant to leave a comment to update the label key as oie.authenticator.webauthn.label'

Reply:
Updated according to our call.